web/src/App.less

```diff
@@ -54,8 +54,8 @@
   background-size: 25px, 25px;
   background-position: center;
   background-repeat: no-repeat;
-  width: 45px;
-  height: 65px;
+  width: 40px;
```

Comment:
Why change this?

Reply:
The height of `<header>` is 64px not 65px